ouroboros-consensus/src/Ouroboros/Consensus/Storage/ChainDB/Impl/Types.hs

```diff
@@ -650,6 +656,9 @@ data TraceAddBlockEvent blk =
     -- updated.
   | PipeliningEvent (TracePipeliningEvent blk)
 
+    -- | Heralds of 'AddedToCurrentChain' or 'SwitchedToAFork'. Lists the tip of the new chain.
+  | SwitchingSelection (Point blk)
```

Comment:
Maybe this should be `ChangingSelection`. What do you think? Is `Switching` too indicative of `SwitchToAFork`?